PennMobile/Dining/SwiftUI/Views/Venue/DiningVenueView.swift

```diff
@@ -59,20 +45,26 @@ struct DiningVenueView: View {
             }
         }
         .listStyle(.plain)
-        .sheet(isPresented: $alertAccepted) {
-            DiningLoginNavigationView(showSheetView: self.$alertAccepted)
-        }
-        .alert(isPresented: $alertIsShowing) {
-            showCorrectAlert()
-        }
     }
 }
 
 struct CustomHeader: View {
 
     let name: String
-    var alertIsShowing: Binding<Bool>?
     var refreshButton = false
+    @State var alertIsShowing = false
```

Comment:
let's try to rename this alert so first time readers can understand why the alert is being thrown something like `showingMissingDiningTokenAlert`